example/settings.py

```diff
@@ -16,6 +16,7 @@
 # Application definition
 
 INSTALLED_APPS = [
+    *(["daphne"] if os.getenv("ASYNC_SERVER", False) else []),
```

Comment:
What do you think about defining an `async_settings.py` file that would contain the async settings such as this one and `ASGI_APPLICATION`? It would change the run command to `python example/manage.py runserver --settings=example.async_settings`

Reply:
FWIW I prefer the single settings module pattern (I'd rather have a few `if` statements than star imports) but I don't feel strongly about it in this project. If different environments have to have different settings at all I find it clearer to see at a glance what the differences are. Maybe that's just me though.

Reply:
I don't have a strong opinion here (use single settings on real-life projects myself but that does not have to apply everywhere else).

Reply:
> FWIW I prefer the single settings module pattern (I'd rather have a few if statements than star imports) but I don't feel strongly about it in this project. If different environments have to have different settings at all I find it clearer to see at a glance what the differences are.

Wouldn't it be easier to see the differences are between environments with the multiple settings modules?

Reply:
@tim-schilling As I wrote above, it's my preference and I don't want to impose it on anyone. I find it easier because I do not have to check several files for the settings, only one file. I don't have to do the "merging" of settings in my head, it's right there in the code in an if statement. But again, just my preference. If you feel strongly about that (for django-debug-toolbar) feel free to request splitting up the settings modules. I'm `-0` on the split, not `-1`.

Reply:
Gotcha. I was confused by your earlier statement. We can keep it as is for now.

Reply:
..and for the record, `example/settings.py` already has a few `if`s :)